packages/lib/src/vis/tiles/TiledHeatmapLayer.tsx

```diff
@@ -0,0 +1,72 @@
+import { Suspense } from 'react';
+import { LinearFilter, NearestFilter, Vector2 } from 'three';
+
+import { useVisibleDomains } from '../hooks';
+import { useAxisSystemContext } from '../shared/AxisSystemContext';
+import Tile from './Tile';
+import type { TilesApi } from './api';
+import type { ColorMapAttributes } from './models';
+import { getTileOffsets, sortTilesByDistanceTo } from './utils';
+
+interface Props extends ColorMapAttributes {
+  api: TilesApi;
+  lod: number;
+}
+
+function TiledHeatmapLayer(props: Props) {
```

Comment:
I would maybe rename this to `TiledLayer` so it's more easily distinguishable from the root component name when searching for files.